docs/user/ovirt/install_ipi.md

```diff
@@ -0,0 +1,77 @@
+# Install using oVirt platform provider
```

Comment:
this should look like other platforms.

Reply:
The other platforms like AWS have a README.md in `docs/user/aws` that has steps that needs to be followed to get IPI install, separate steps like Setup DNS, Permissions, Install cluster etc.

Reply:
I think this is following the KNI doc: https://github.com/openshift/installer/blob/master/docs/user/metal/install_ipi.md